src/WPF/WPF.Viewer/MainWindow.xaml.cs

```diff
@@ -326,13 +362,29 @@ private void PopulateSearchedTree()
             if (!string.IsNullOrWhiteSpace(SearchFilterBox.SearchText))
             {
                 OpenCategoryLeaves();
+                TrackSearchEvent();
```

Comment:
I am concerned about an overactive search reporting here. I do like this better than grouping the search info with other event reports. I guess we won't know until we see it in action.

Reply:
Good point thank you, I think there are a few things we can do if the activity is too high:

- We can increase the delay time to prevent tracking searches before the user has finished typing
- We can batch the search events after capturing full terms 
- Other options can be considered 

I think it's something for us to visit in the future once we have seen some data come in.